shell/components/SlideInPanelManager.vue

```diff
@@ -23,8 +28,44 @@ const panelTop = computed(() => {
 const panelHeight = computed(() => `calc(100vh - ${ panelTop?.value })`);
 const panelWidth = computed(() => currentProps?.value?.width || '33%');
 const panelRight = computed(() => (isOpen?.value ? '0' : `-${ panelWidth?.value }`));
+const panelZIndex = computed(() => `${ (isOpen?.value ? 1 : 2) * (currentProps?.value?.zIndex ?? 1000) }`);
 
-const panelTitle = computed(() => currentProps?.value?.title || 'Details');
+const showHeader = computed(() => currentProps?.value?.showHeader ?? true);
+const panelTitle = showHeader.value ? computed(() => currentProps?.value?.title || 'Details') : null;
+
+watch(
```

Comment:
I gave this a test and works correctly. Might need some tweaking in light of my previous comment where I advise to keep the focus trap always on (no flag needed).

On `PromptModal` we trigger automatically the focus trap on `AppModal` (always true) and I pass a the "variable to watch" that is dependent on wether the component is rendered or not https://github.com/rancher/dashboard/blob/master/shell/components/PromptModal.vue#L125

I think it might work for you here as well

Reply:
Yeah, the watcher is a good idea. I'll remove the `triggerFocusTrap` as well so that it will be always enabled for the slide-in. Thanks